src/_includes/layouts/base.html

```diff
@@ -163,6 +160,11 @@
     <script src="{{ '/assets/js/tabs.js' | url }}"></script>
     <script src="{{ '/assets/js/components-index.js' | url }}"></script>
     {%- endif -%}
+    <script>
```

Comment:
Just wondering why we'd need this piece of code? Are we applying any special style or using it somewhere? else this is totally un necessary?

Reply:
Seems like there is no special styling exists as of now and not able to find where this is being used also.

Reply:
I guess we don't need this now. Previously this logic was added to set specific `font-family` for windows. After some time that styling code was removed.

Reply:
Shall I remove the userAgent check, then?

Reply:
@kecrily Can you confirm this?

Reply:
This is needed to fix an issue with emoji flags not showing up in Chromium on Windows:
https://github.com/eslint/eslint.org/issues/183

Reply:
I don’t see that #213 makes use of the class win to show emojis? Or am I missing something 😅

Reply:
Yes. It was used to restrict twemoji to Windows and use the default emoji for other platforms. I was later told that twemoji should be used for all platforms. This code seems useless now, let's remove it.

Reply:
> I don’t see that #213 makes use of the class win to show emojis? Or am I missing something 😅

It's in #215

Reply:
Yes, if it's not used let's remove 👍🏻